lib/model/store.dart

```diff
@@ -793,6 +793,11 @@ class PerAccountStore extends PerAccountStoreBase with
       case UserGroupEvent():
         assert(debugLog("server event: user_group/${event.op}"));
         _groups.handleUserGroupEvent(event);
+        if (event is UserGroupRemoveEvent) {
+          autocompleteViewManager.handleUserGroupRemoveEvent(event);
+        } else if (event is UserGroupUpdateEvent) {
+          autocompleteViewManager.handleUserGroupUpdateEvent(event);
+        }
```

Comment:
```
store: Call AutocompleteViewManager.handleUserGroupRemove/UpdateEvent
```

Oh, thanks for the catch!

I guess this is a bugfix where @-mention autocomplete would misbehave after a user-group was renamed or removed in between @-mention attempts.